### DISCUSSION TITLE
quests/parking_lanes

### DISCUSSION THREAD

**Discussion by TurnrDev** [streetcomplete/StreetComplete#2993] [General]
In the source code, I see `quests/parking_lanes/ParkingLane.kt` and `quests/parking_lanes/ParkingLanesParser.kt`. But I'm not sure where it's used. I don't see a parking lanes quest. Just seeking guidance here :)

**Comment by matkoniecz** [comment]
Looks like unfinished work in progress on parking lane quest.
